ssg/build_remediations.py

```diff
@@ -25,7 +25,7 @@
 FILE_GENERATED_HASH_COMMENT = '# THIS FILE IS GENERATED'
 
 REMEDIATION_CONFIG_KEYS = ['complexity', 'disruption', 'platform', 'reboot',
```

Comment:
I don't like this so much - this PR is mixing remediation metadata and rule data. I would prefer re-reading the rule YAML from the processed rules directory, or to introduce the rule title as a new metadata attribute.

Reply:
Aha cool